src/Makefile

```diff
@@ -58,63 +60,17 @@ clean :
 
 sieve.o : sieve.c
 	$(CC) $(CFLAGS) $(CFLAGS_EXTRA_SIEVE) -c $< -o $@
-	
+
 tf_75bit.o : tf_96bit.cu
 	$(NVCC) $(NVCCFLAGS) -c $< -o $@ -DSHORTCUT_75BIT
-	
+
 %.o : %.cu
 	$(NVCC) $(NVCCFLAGS) -c $< -o $@
 
 %.o : %.c
 	$(CC) $(CFLAGS) -c $< -o $@
 
-##############################################################################
-
-# dependencies generated by cpp -MM
-checkpoint.o: checkpoint.c params.h
-
-# manually add selftest-data-mersenne.c or selftest-data-wagstaff.c
-mfaktc.o: mfaktc.c params.h my_types.h compatibility.h sieve.h \
- read_config.h parse.h timer.h tf_72bit.h tf_96bit.h tf_barrett96.h \
- checkpoint.h signal_handler.h output.h gpusieve.h \
- selftest-data-mersenne.c selftest-data-wagstaff.c
-
-output.o: output.c params.h my_types.h output.h compatibility.h
-
-parse.o: parse.c compatibility.h parse.h params.h
-
-read_config.o: read_config.c params.h my_types.h
-
-sieve.o: sieve.c params.h compatibility.h
-
-signal_handler.o: signal_handler.c params.h my_types.h compatibility.h
-
-timer.o: timer.c timer.h compatibility.h
-
-tf_72bit.o: tf_72bit.cu params.h my_types.h compatibility.h \
- my_intrinsics.h sieve.h timer.h output.h tf_debug.h tf_common.cu
-
-tf_96bit.o: tf_96bit.cu params.h my_types.h compatibility.h \
- my_intrinsics.h sieve.h timer.h output.h tf_debug.h \
- tf_96bit_base_math.cu tf_96bit_helper.cu gpusieve_helper.cu tf_common.cu \
- tf_common_gs.cu gpusieve.h
-
-tf_barrett96.o: tf_barrett96.cu params.h my_types.h compatibility.h \
- my_intrinsics.h sieve.h timer.h output.h tf_debug.h \
- tf_96bit_base_math.cu tf_96bit_helper.cu tf_barrett96_div.cu \
- tf_barrett96_core.cu tf_common.cu
-
-tf_barrett96_gs.o: tf_barrett96_gs.cu params.h my_types.h compatibility.h \
- my_intrinsics.h sieve.h timer.h output.h tf_debug.h \
- tf_96bit_base_math.cu tf_96bit_helper.cu tf_barrett96_div.cu \
- tf_barrett96_core.cu gpusieve_helper.cu tf_common_gs.cu gpusieve.h
-
-gpusieve.o: gpusieve.cu params.h my_types.h compatibility.h \
- my_intrinsics.h gpusieve.h
-
-# manually generated dependency
+../%.ini : %.ini
+	$(CP) $< ..
 
-tf_75bit.o: tf_96bit.cu params.h my_types.h compatibility.h \
- my_intrinsics.h sieve.h timer.h output.h tf_debug.h \
- tf_96bit_base_math.cu tf_96bit_helper.cu gpusieve_helper.cu tf_common.cu \
- tf_common_gs.cu gpusieve.h
+.PHONY: all clean
```

Comment:
Please add the final newline. This also applies to other files where the red circle appears.